src/Hint/NewType.hs

```diff
@@ -27,7 +28,14 @@ newtype Foo = Foo Int deriving stock Show
 module Hint.NewType (newtypeHint) where
 
 import Hint.Type
+-- TODO: remove these qualifieds
+import qualified "ghc-lib-parser" HsDecls as Hs
+import qualified "ghc-lib-parser" HsTypes as Hs
+import qualified "ghc-lib-parser" SrcLoc  as Hs
 
+-- TODO: Adapt to use ghc-lib-parser:
+-- * figure out pretty printing ???
```

Comment:
See https://github.com/ndmitchell/hlint/blob/master/src/Idea.hs#L90 - call `idea'` and it will pretty print for you.